streams/src/test/java/org/apache/kafka/streams/state/internals/SessionBytesStoreTest.java

```diff
@@ -593,7 +572,7 @@ public void shouldNotThrowInvalidRangeExceptionWithNegativeFromKey() {
         return results;
     }
 
-    protected static <K, V> Set<KeyValue<K, V>> toSet(final Iterator<KeyValue<K, V>> iterator) {
+    private static <K, V> Set<KeyValue<K, V>> toSet(final Iterator<KeyValue<K, V>> iterator) {
```

Comment:
Actually, doesn't HashSet have a constructor to do this, now?

Reply:
Could you point me to it? I thought you can only do a `(Collection) iterable` as the param of the constructor, but for iterator I'm not aware of a way.